nightfall-deployer/circuits/withdraw.zok

```diff
@@ -11,7 +11,7 @@ def main(\
     PublicTransaction tx,\
     field[2] roots,\
     private Nullifiers<2> nullifiers,\
-    private Commitments<2> commitments\
+    private Commitments<1> commitments\
```

Comment:
If we set Commitments<1> in the withdraw.zok, withdraw_stub.zok needs to also be updated